database-service/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+addopts =
+    -p no:cacheprovider
```

Comment:
This is one of the necessary hacks to prevent `pytest` from producing cached data. Those data are generated by the root user and cannot be deleted anymore. Another workaround would be to run the `docker-compose` command of the tests as user `$(id -u)`, as is done for fixtures generation.

Reply:
not yet, maybe later when we have more tests; we could run the tests as user ˋid -uˋ, that would solve that problem. As I said already, the issue is that the jenkins agent running the pipeline doesn't have the permissions to remove or modify the cache folder generated by a previous test run as it was generated by the root user of the test docker container.